### ISSUE TITLE
odoo.com/forum page stop responding and cause 100% chrome CPU, after I remove the <header> element

### ISSUE THREAD

**Issue by odoochina-dev** [odoo/odoo#65151]
Impacted versions:
latest odoo 14 used at odoo.com/forum

My env:
 - Windows 10, version: 19042.746 (almost latest)
 - Chrome,  87.0.4280.141 (almost latest)
   (btw, Chromium based Edge 86.0.622.63 does NOT have this issue...)

Steps to reproduce:
1) open a page in odoo.com/forum. e.g. this one:
https://www.odoo.com/forum/help-1/how-can-i-filter-for-empty-non-empty-one-to-many-relations-83221
2) open chrome DevTools -> Elements tab, find the 'body > div#wrapwrap > header#top' element, press delete to remove it from DOM.  (see attached pic1)
3) put mouse focus on this webpage, and wheel the mouse to top of the page, and then scroll down. (see attached pic2)

Current behavior:
This chrome tab get not responding and chrome task manager shows it keeping consuming 100% CPU (until the tab get crashed)

Expected behavior:
shall not NO responding and shall not consuming so much CPU.

Video/Screenshot link (optional):
step1:
![odoo-bug-1](https://user-images.githubusercontent.com/76577822/106015048-0ca20500-60f9-11eb-8352-911de8ba51d7.png)
step2-3:
![odoo-bug-2](https://user-images.githubusercontent.com/76577822/106015059-0f9cf580-60f9-11eb-9bc3-7ce20736727c.png)

Support ticket number submitted via odoo.com/help (optional):

**Comment by Yenthe666** [comment]
Euhh, why would you ever do this though? Doesn't make sense to me. CC @pedrobaeza

**Comment by odoochina-dev** [comment]
> Euhh, why would you ever do this though? Doesn't make sense to me. CC @pedrobaeza

well, I by happen found it. And, yes, It's not a fault from normal functionality pov, but, I believe here must be some incorrect event handling (kind of dead loop), either bug of odoo, or owl framework? I'm not sure.

**Comment by pedrobaeza** [comment]
This is something that you don't get with normal usage, but forcing a removal of a DOM element, that doesn't happen on regular use. It's the same as programming something to make it fail. It can be considered a bug.

If you find something reproducible without manipulating things, please report back. Meanwhile, I close this.

**Comment by odoochina-dev** [comment]
No, I believe remove element from DOM or add new element to DOM is quite normal, that's why jQuery have .remove(), .detach(), .append(), ... etc., this kind of APIs.
And what ever kind of operation, it shall not cause 100% CPU... IMHO.

But, well, you can close it as it's not normal end user case from odoo/odoo pov. I'll try to open another one at odoo/owl to see whether it's bug of owl.
